src/libraries/System.Private.CoreLib/src/System/String.Searching.cs

```diff
@@ -423,6 +448,104 @@ public int LastIndexOf(char value, int startIndex, int count)
             return result < 0 ? result : result + startSearchAt;
         }
 
+        /// <summary>
+        /// Reports the zero-based index of the last occurrence of the specified character in the current String object.
+        /// A parameter specifies the type of search to use for the specified character.
+        /// </summary>
+        /// <param name="value">The character to seek.</param>
+        /// <param name="comparisonType">One of the enumeration values that specifies the rules for the search.</param>
+        /// <returns>
+        /// The zero-based index position of <paramref name="value"/> from the end of the current instance
+        /// if that character is found, or -1 if it is not.
+        /// </returns>
+        public int LastIndexOf(char value, StringComparison comparisonType)
+        {
+            return LastIndexOf(value, Length - 1, comparisonType);
+        }
```

Comment:
should we check for empty string first? This can throw exception later if this is the case.